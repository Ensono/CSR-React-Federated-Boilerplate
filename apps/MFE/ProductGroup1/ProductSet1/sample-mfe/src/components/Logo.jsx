import React from 'runtime-library/lib/react'
import pictureData from './MF.jpeg'

export default function () {
  return <img src={pictureData} style={{width: '500px', borderRadius: '10px'}} />
}
