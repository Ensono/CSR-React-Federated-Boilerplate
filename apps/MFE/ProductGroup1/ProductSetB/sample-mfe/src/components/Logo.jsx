import React from "runtime-library/lib/react"
import pictureData from "./MF.jpeg"

export default function () {
  return <img alt="test" src={pictureData} style={{width: "500px", borderRadius: "10px"}} />
}
