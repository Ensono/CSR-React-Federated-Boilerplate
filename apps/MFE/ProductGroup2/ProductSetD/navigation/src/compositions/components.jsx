import styled from "runtime-library/lib/@emotion/styled"

export const Header = styled.div`
  background-color: #f4dbca;
  padding: 10px 16px;
  border: 1px solid #dcdfe6;
  display: flex;
  cursor: pointer;
`
export const Link = styled.li`
  text-decoration: none;
  font-size: 1.5rem;
`
