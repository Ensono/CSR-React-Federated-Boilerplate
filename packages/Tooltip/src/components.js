import styled from '@emotion/styled';

export const TooltipComponent = styled.div`
    background-color: #fff;
    padding: 10px 16px;
    border: 1px solid #dcdfe6;
    display: inline-block;
    cursor: pointer;
    border-radius: 4px;
    position: relative;
    &:before {
        content: attr(data-content);
        max-width: 100%;
        box-sizing: border-box;
        position: absolute;
        background-color: #303133;
        color: #fff;
        font-size: 12px;
        border-radius: 4px;
        padding: 10px;
        left: 50%;
        bottom:100%;
        transform: translate(-50%,-10px);
        display: none;
    }
    &:after{
        display: none;
        content: "";
        border: 6px solid transparent;
        border-top-color:#303133;
        position: absolute;
        left: 50%;
        bottom:100%;
        transform: translate(-50%,2px);
    }
    &:hover::after, &:hover::before {
        display: block;
    }
`;
