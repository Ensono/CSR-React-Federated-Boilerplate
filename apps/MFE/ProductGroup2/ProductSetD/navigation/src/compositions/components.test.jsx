import React from "runtime-library/lib/react"
import {render} from "@testing-library/react"
import {Header} from "./components"

describe("Header: ", () => {
  it("Header should match the snapshot for type = primary", () => {
    const {asFragment} = render(<Header />)
    expect(asFragment()).toMatchSnapshot()
  })
})
