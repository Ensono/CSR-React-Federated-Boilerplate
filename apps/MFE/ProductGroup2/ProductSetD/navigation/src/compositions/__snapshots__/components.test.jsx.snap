// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header:  Header should match the snapshot for type = primary 1`] = `
<DocumentFragment>
  .emotion-0 {
  background-color: #f4dbca;
  padding: 10px 16px;
  border: 1px solid #dcdfe6;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
}

<div
    class="emotion-0"
  />
</DocumentFragment>
`;
